src/main/java/com/google/maps/DistanceMatrixApiRequest.java

```diff
@@ -25,9 +23,10 @@
 import com.google.maps.model.TransitRoutingPreference;
 import com.google.maps.model.TravelMode;
 import com.google.maps.model.Unit;
-
 import org.joda.time.ReadableInstant;
 
+import static com.google.maps.internal.StringJoin.join;
```

Comment:
Going by the Google style guide, this was correct before. This is in a few places.